src/projects/gt4o_us/setting/setting.ad

```diff
@@ -0,0 +1,421 @@
+module SettingProject
+{
+    #include "../share/dialog.ad";
+	#include "../share/page_util.ad";
+
+	static BRAKE_list = [
+	"BRAKE_NORMAL",
+	"RACING_BRAKE"
+	];
+	
+	static BRAKECONTROLLER_list = [
+	"NO_BRAKECONTROLLER",
+	"BRAKECONTROLLER"
+	];
+	
+	static SUSPENSION_list = [
+		"NORMAL_SUSPENSION",
+		"SPORTS_SUSPENSION",
+		"SEMIRACING_SUSPENSION",
+		"RACING_SUSPENSION",
+		"ORIGINAL_SUSPENSION"
+	];
+	
+	static ASCC_list = ["NO_ASCC", "ASCC"];
+	
+	static TCSC_list = ["NO_TCSC", "TCSC"];
+	
+	static CHASSIS_list = [nil];
+	
+	static RACINGMODIFY_list = [nil];
+	
+	static LIGHTWEIGHT_list = [
+	nil,
+	"LIGHT_WEIGHT_STAGE1",
+	"LIGHT_WEIGHT_STAGE2",
+	"LIGHT_WEIGHT_STAGE3"
+	];
+	
+	static STEER_list = [nil];
+	
+	static DRIVETRAIN_list = ["NORMAL_DRIVETRAIN", "VALIABLE_CENTER_DEF_DRIVETRAIN"];
+	
+	static GEAR_list = [
+	"NORMAL_GEAR",
+	"CLOSE_GEAR",
+	"SUPER_CLOSE_GEAR",
+	"VALIABLE_GEAR"
+	];
+	
+	static ENGINE_list = [nil];
+	
+	static NATUNE_list = [
+	nil,
+	"NA_TUNE_STAGE1",
+	"NA_TUNE_STAGE2",
+	"NA_TUNE_STAGE3"
+	];
+	
+	static TURBINEKIT_list = [
+		"NO_TURBINE_KIT",
+		"TURBINE_KIT_STAGE1",
+		"TURBINE_KIT_STAGE2",
+		"TURBINE_KIT_STAGE3",
+		"TURBINE_KIT_STAGE4",
+		"TURBINE_KIT_STAGE5"
+	];
+	
+	static PORTPOLISH_list = [
+	nil,
+	"PORTPOLISH"
+	];
+	
+	static ENGINEBALANCE_list = [nil, "ENGINEBALANCE"];
+	
+	static DISPLACEMENT_list = [nil, "DISPLACEMENT"];
+	
+	static COMPUTER_list = [nil, "COMPUTER"];
+	
+	static INTERCOOLER_list = [
+	nil,
+	"INTERCOOLER_SPORTS",
+	"INTERCOOLER_RACING_BIG"
+	];
+	
+	static MUFFLER_list = [
+	nil,
+	"SPORTS_MUFFLER",
+	"SEMI_RACING_MUFFLER",
+	"RACING_MUFFLER"
+	];
+	
+	static CLUTCH_list = [
+		"NORMAL_CLUTCH",
+		"HIGH_CAPACITY_CLUTCH",
+		"TWIN_CLUTCH",
+		"TRIPLE_CLUTCH"
+	];
+	
+	static FLYWHEEL_list = [
+	nil,
+	"FLYWHEEL_LIGHT",
+	"FLYWHEEL_Cr_Mo",
+	"FLYWHEEL_LIGHT_Cr_Mo"
+	];
+	
+	static PROPELLERSHAFT_list = [nil, "PROPELLERSHAFT"];
+	
+	static LSD_list = [
+		"NORMAL_LSD",
+		"1WAY_LSD",
+		"15WAY_LSD",
+		"2WAY_LSD",
+		"VARIABLE_LSD",
+		"AYCC_LSD"
+	];
+	
+	static FRONTTIRE_list = [
+		"FRONTTIRE_NORMAL_ECONOMY",
+		"FRONTTIRE_NORMAL_COMFORT",
+		"FRONTTIRE_NORMAL_STREET",
+		"FRONTTIRE_SPORTS_HARD",
+		"FRONTTIRE_SPORTS_MEDIUM",
+		"FRONTTIRE_SPORTS_SOFT",
+		"FRONTTIRE_RACING_SUPER_HARD",
+		"FRONTTIRE_RACING_HARD",
+		"FRONTTIRE_RACING_MEDIUM",
+		"FRONTTIRE_RACING_SOFT",
+		"FRONTTIRE_RACING_QUALIFY",
+		"FRONTTIRE_DIRT",
+		"FRONTTIRE_SNOW"
+	];
+	static REARTIRE_list = [
+		"REARTIRE_NORMAL_ECONOMY",
+		"REARTIRE_NORMAL_COMFORT",
+		"REARTIRE_NORMAL_STREET",
+		"REARTIRE_SPORTS_HARD",
+		"REARTIRE_SPORTS_MEDIUM",
+		"REARTIRE_SPORTS_SOFT",
+		"REARTIRE_RACING_SUPER_HARD",
+		"REARTIRE_RACING_HARD",
+		"REARTIRE_RACING_MEDIUM",
+		"REARTIRE_RACING_SOFT",
+		"REARTIRE_RACING_QUALIFY",
+		"REARTIRE_DIRT",
+		"REARTIRE_SNOW"
+	];

```

Comment:
This is less matching than previously; here's a hint of how it should look like
In other words you didn't need to wrap the elements into arrays

```
  // static BRAKE_list = Array(2);
  1EE5|   5| 69| STATIC_DEFINE: BRAKE_list
  1EF6|   5| 70| VARIABLE_PUSH: BRAKE_list,BRAKE_list, Static:1
  1F1B|   5| 71| VARIABLE_EVAL: Array,Array, Static:2
  1F36|   5| 72| INT_CONST: 2 (0x02)
  1F3F|   5| 73| CALL: ArgCount=1
  1F48|   5| 74| EVAL
  1F4D|   5| 75| ASSIGN_OLD
  1F52|   5| 76| POP_OLD

  // BRAKE_list[0] = "BRAKE_NORMAL"
  1F57|   6| 77| VARIABLE_EVAL: BRAKE_list,BRAKE_list, Static:1
  1F7C|   6| 78| INT_CONST: 0 (0x00)
  1F85|   6| 79| BINARY_OPERATOR: [] ([])
  1F8E|   6| 80| STRING_CONST: BRAKE_NORMAL
  1FA1|   6| 81| ASSIGN_OLD
  1FA6|   6| 82| POP_OLD
```